executor/builder.go

```diff
@@ -617,17 +619,6 @@ func (b *executorBuilder) buildDDL(v *plan.DDL) Executor {
 	return e
 }
 
-func (b *executorBuilder) buildExplain(v *plan.Explain) Executor {
```

Comment:
Move code makes a reviewer waste more time to view a PR

Reply:
Yup. Thanks for the kind and good advice. Already did this, please refer to #7065 and #7063.